CefSharp.WinForms/Internals/MovingListener.cs

```diff
@@ -0,0 +1,155 @@
+﻿// Copyright © 2014 The CefSharp Authors. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style license that can be found in the LICENSE file.
+
+using CefSharp.Internals;
+using System;
+using System.Drawing;
+using System.Runtime.InteropServices;
+using System.Windows.Forms;
+
+namespace CefSharp.WinForms.Internals
+{
+    internal class MovingListener : NativeWindow
```

Comment:
I'm not sure what you're saying here. (probably because I don't have enough context)

Are you asking me to change the class name from `MovingListener` to `BrowserParentFormWindowsMessageListener` or something similar so the name is a bit more generic?

If so, that's fine by me, let me have a name suggestion. :)